airflow/bin/cli.py

```diff
@@ -1842,6 +1842,13 @@ def sync_perm(args): # noqa
         print('The sync_perm command only works for rbac UI.')
 
 
+@cli_utils.action_logging
+def sync_to_db(args): # noqa
+    dagbag = DagBag(settings.DAGS_FOLDER, store_serialized_dags=settings.STORE_SERIALIZED_DAGS)
```

Comment:
Does Dagbag have sync_to_db method? In Airflow 2.0, we have it. In Airflow 1.10 it doesn't exist yet. How is it in this case?

Reply:
doesn't exist yet. I could port it over as well, but it does basically the same thing.